content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -0,0 +1,308 @@
+[feature](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/20190228-priority-and-fairness.md),
+which is not enabled by default. To enable it, add the following command-line flags
+to your `kube-apiserver` invocation:
+
+```shell
+kube-apiserver \
+--feature-gates=APIPriorityAndFairness=true \
+--runtime-config=flowcontrol.apiserver.k8s.io/v1alpha1=true \
+ # …and other flags as usual
+```
+
+## Concepts
+There are several distinct features involved in the flow control API. Incoming
+requests are classified by attributes of the request using _FlowSchemas_, and
+assigned to priority levels. Priority levels add a degree of isolation by
+maintaining separate concurrency limits, so that requests assigned to different
+priority levels cannot starve each other. Within a priority level, a
+fair-queuing algorithm prevents requests from different sources from starving
+each other, and allows for requests to be queued to prevent bursty traffic from
+causing failed requests when the average load is acceptably low.
+
+### Priority Levels
+Without the Priority and Fairness feature enabled, overall concurrency in the
+API server is limited by the `kube-apiserver` flags `--max-requests-inflight`
+and `--max-mutating-requests-inflight`. With the feature enbaled, the
+concurrency limit defined by these flags is divided up among a configurable set
+of _priority levels_. Incoming requests are assigned to a single priority level,
+and each priority level will only dispatch as many concurrent requests as its
+configuration allows.
+
+The default configuration, for example, includes separate priority levels for
+leader-election requests, requests from built-in {< glossary_tooltip text="controllers" term_id="controller" >}}, and requests from
+Pods. This means that an ill-behaved Pod that floods the API server with
+requests cannot prevent leader election or actions by the built-in controllers
+from succeeding.
+
+### Fair Queuing
+Even within a priority level there may be a large number of distinct sources of
+traffic. In an overload situation, it is valuable to prevent one consumer from
+starving the others. This is handled by use of a fair-queuing algorithm to
+process requests that are assigned the same priority level. Each request is
+given a _flow distinguisher_ — in the current design, this is either the
+requesting user or the resource's namespace — and the system attempts to give
+approximately equal weight to requests with different flow distinguishers.
+
+The details of the queuing algorithm are tunable for each priority level, and
+allow administrators to trade off memory use against fairness and tolerance for
+bursty traffic.
+
+### Exempt requests
+Some requests are considered sufficiently important that they are not subject to
+any of the limitations imposed by this feature. This prevents an
+improperly-configured flow control configuration from totally disabling an API
+server.
```

Comment:
and those configurations for exempt requests are mandatory and not supposed to be modifier by the cluster administrators